explainaboard/metrics/accuracy.py

```diff
@@ -46,6 +66,86 @@ def calc_stats_from_data(
             )
         )
 
+    def _calc_metric_from_aggregate(
+        self,
+        agg_stats: np.ndarray[tuple[int], Any] | np.ndarray[tuple[int, int], Any],
+    ) -> np.ndarray[tuple[()], Any] | np.ndarray[tuple[int], Any]:
+        """Inner function of calc_metric_from_aggregate."""
+        if agg_stats.shape[-1] == 1:
+            return agg_stats.squeeze(-1)
+        return agg_stats
```

Comment:
I think you intended to return both the score and confidence from this function, but the caller of this function allows to return only the score. You need to implement another function to return the confidence scores.